tests/affine-cloud-copilot/e2e/chat-with/text.spec.ts

```diff
@@ -62,13 +62,16 @@ test.describe('AIChatWith/Text', () => {
     loggedInPage: page,
     utils,
   }) => {
-    const { translate } = await utils.editor.askAIWithText(page, 'Apple');
-    const { answer } = await translate('German');
-    await expect(answer).toHaveText(/Apfel/, { timeout: 10000 });
+    const { fixSpelling } = await utils.editor.askAIWithText(
+      page,
+      'I Loev Apple'
+    );
```

Comment:
Select only `Loev` then fix spelling?

Reply:
Let me c.... how to select specific text within a paragraph.

Reply:
Playwright currently can't handle this